kommunicate/src/main/java/io/kommunicate/models/KmAppSettingModel.java

```diff
@@ -0,0 +1,138 @@
+package io.kommunicate.models;
+
+import com.applozic.mobicommons.json.JsonMarker;
+
+public class KmAppSettingModel extends JsonMarker {
+    private String code;
+    private KmResponse response;
+    public static final String SUCCESS = "SUCCESS";
+
+    public String getCode() {
+        return code;
+    }
+
+    public void setCode(String code) {
+        this.code = code;
+    }
+
+    public KmResponse getResponse() {
+        return response;
+    }
+
+    public void setResponse(KmResponse response) {
+        this.response = response;
+    }
+
+    public boolean isSuccess() {
+        return SUCCESS.equals(getCode());
+    }
+
+    public KmChatWidget getChatWidget() {
+        return response.getChatWidget();
+    }
+
+    public static class KmResponse extends JsonMarker {
+        private String userName;
+        private String agentId;
+        private String agentName;
+        private KmChatWidget chatWidget;
+
+        public String getUserName() {
+            return userName;
+        }
+
+        public void setUserName(String userName) {
+            this.userName = userName;
+        }
+
+        public String getAgentId() {
+            return agentId;
+        }
+
+        public void setAgentId(String agentId) {
+            this.agentId = agentId;
+        }
+
+        public String getAgentName() {
+            return agentName;
+        }
+
+        public void setAgentName(String agentName) {
+            this.agentName = agentName;
+        }
+
+        public KmChatWidget getChatWidget() {
+            return chatWidget;
+        }
+
+        public void setChatWidget(KmChatWidget chatWidget) {
+            this.chatWidget = chatWidget;
+        }
+    }
+
+    public class KmChatWidget extends JsonMarker {
+        private boolean popup;
+        private Short iconIndex;
+        private String primaryColor;
+        private String secondaryColor;
+        private boolean showPoweredBy;
+        private long sessionTimeout;
+        private long botMessageDelayInterval;
```

Comment:
it is possible that customer havn't configured it so this can be null, same with sessionTimeout.
safe side, it will be better to use wrapper class.
Also, these 2 value would never be that big to come under long.

Reply:
I guess sessionTimeout and botMessageDelay are not required on mobile side and are neither used currenlty. We can remove these properties.

Reply:
they will be used in later versions, we can keep them

Reply:
Yes, if those woudnt be timestamp then we can change them to int.

Reply:
it will be integers, 
bot time delay is in set in seconds on dashboard
and no one will set it to be more than 15-20 seconds. Standard value for this is 1 second typing time.
trying setting the value here: https://dashboard.kommunicate.io/settings/chat-widget-customization#bot-typing-indicator
and
https://dashboard.kommunicate.io/settings/chat-widget-customization#chat-widget-security
to see the response value coming.
check in both cases, one is a new signup without modifying any value. another after setting some value.

Reply:
Done